src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let res = 0;
+
+  for (let index = 0; index < days; index++) {
+    res += 40;
+  }
+
+  if (days < 3) {
+    return res;
+  } else if (days >= 3 && days < 7) {
```

Comment:
If you use `return` inside `if` block, using `else` is useless.
This is the correct structure:
```
if (condition1) {
  return x;
}

if (condition2) {
  return y
}

return z;
```